Packages/Settings/Sources/SettingsUI/Internal/GitHubSettings/GitHubPrivateKeyPicker.swift

```diff
@@ -12,6 +12,8 @@ struct GitHubPrivateKeyPicker: View {
             L10n.Settings.Github.PrivateKey.Scopes.organization
         case .repo:
             L10n.Settings.Github.PrivateKey.Scopes.repository
+        case .enterpriseServer:
+            "Check permissions: `manage_runners:enterprise`"
```

Comment:
This text needs to be localized and formatted similar to the other texts listing required permissions.